uk.co.compucorp.civicrm.hrcore/gulpfile.js

```diff
@@ -46,6 +47,31 @@ var Promise = require('es6-promise').Promise;
     });
   });
 
+  /**
+   * Gets a map of contact roles and IDs
+   *
+   * @return {Promise} resolved with {Object}
+   */
+  function getRolesAndIDs () {
+    return new Promise(function (resolve) {
+      var rolesAndIDs = {};
```

Comment:
this should be declared inside the `exec` callback, it has no purpose here

Reply:
Done.